packages/migrator/src/index.ts

```diff
@@ -77,17 +77,21 @@ export const defaultResolvers = {
 export const setupSlonikMigrator = ({
   slonik,
   migrationsPath,
-  migrationTableName = 'migration',
+  migrationTableName = ['public', 'migration'],
```

Comment:
Any reason to change the default? Wouldn't this affect the behaviour in cases where the "default" schema _isn't_ `public`?

Reply:
Hmm, this might be a left-over from our own patch. I'll push a change that sets it back to just `migration` when I have the time.

Reply:
@mmkal  I reverted it back to the old name.